website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -826,7 +826,7 @@ The following relationships may be present in various responses.
 
 | Resource Name         | Description                                                                                                                   |
 | --------------------- | ----------------------------------------------------------------------------------------------------------------------------- |
-| `module-producers`    | Other organizations that are configured to share modules with the organization. Terraform Enterprise v202103-1 or later only. |
+| `module-producers`    | Other organizations that are configured to share modules with the organization. |
```

Comment:
```suggestion
| `module-producers`    | Other organizations configured to share modules with the organization. |
```

Suggest changing "organizations that are configured" to "organizations configured" to avoid passive voice.